README.md

```diff
@@ -1,7 +1,124 @@
-# vscode-granite
+<img src="/media/title.png" alt="ollamacontine" width="1000"/>
+
+<!-- ABOUT THE PROJECT -->
+## About The Project
+In this project,we are building local AI co-pilot using **Granite Code, Ollama, and Continue**,where we utilize a collection of open-source components to implement a feature-rich developer co-pilot in Visual Studio Code, all while ensuring data privacy and compliance with licensing requirements. We are integrating Ollama with Continue to seamlessly work with Visual Studio Code. This integration will assist developers by accelerating the development process, enhancing performance, and improving code quality efficiently.
+
+<p align="center">
+  <img src="/media/ollamacontinue.png" alt="Continue" style="vertical-align: middle; width: 800px;" />
+</p>
+
+
+
+## vscode-granite
 
 This extension allows you to easily setup the [Continue extension](https://marketplace.visualstudio.com/items?itemName=Continue.continue) to use [IBM](https://www.ibm.com/)'s [Granite code models](https://github.com/ibm-granite/granite-code-models), as your code assistant in VS Code.
 
+### **Reasons to Choose Ollama**
+
+- **Data Privacy:** Many corporations have privacy regulations that prohibit sending internal code or data to third-party services.
+- **Generated Material Licensing:** Many models, even those with permissive usage licenses, do not disclose their training data and therefore may produce output that is derived from training material with licensing restrictions.
+- **Cost:** Many of these tools are paid solutions that require investment by the organization. For larger organizations, this would often include paid support and maintenance contracts, which can be extremely costly and slow to negotiate.
+
+### Models and Workflow of Vscode-Granite
+
+vscode-granite uses the Granite Code model, which is optimized for enterprise software development workflows and performs well across a range of coding tasks (e.g., code generation, fixing, and explanation), making it a versatile "all-around" code model.
+
+Granite Code comes in a wide range of sizes to fit your workstation's available resources. Generally, the bigger the model, the better the results.
+
+**Recommendation:** Model Size 20B for chat, 8B for tab code completion.
+
+For more details, refer to [Granite Code Models](https://github.com/ibm-granite/granite-code-models)
+
+ ### Workflow of Vscode-Granite
+
+ <p align="center">
+<img src="/media/workflow.png" alt="workflow" width="400"/>
+ </p>
+ 
+### Installation Prerequisites:
+
+  * OS: MacOS 
+  * DISK SPACE :Minimum 30 GB 
+  * latest [Visual Studio Code](https://code.visualstudio.com/)
+  * [Node.js](https://nodejs.org/) v20.0.0 or higher
+  * optional: [gh](https://cli.github.com/) to manage pull requests.
+  * Install [continu.dev](https://marketplace.visualstudio.com/items?itemName=Continue.continue) extention on Visual studio code
+
```

Comment:
Until the extension is available in the Marketplaces, you can explain how you can download the vsix from the GH releases page, and how to install it.